routing/cross_mwm_connector_serialization.hpp

```diff
@@ -47,14 +75,29 @@ class CrossMwmConnectorSerializer final
     }
 
     template <class Sink>
-    void Serialize(serial::CodingParams const & codingParams, uint32_t bitsPerOsmId,
+    void WriteCrossMwmId(connector::OsmId const & id, uint8_t bits, BitWriter<Sink> & w) const
+    {
+      w.WriteAtMost64Bits(id.Get(), bits);
```

Comment:
It would be nice to assert here that bits <= 64.

Reply:
Done